pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -673,6 +681,14 @@ class Build {
                         }
                     }
 
+                    if ("${targetTests}" == 'dev.jck' && "${platform}" == 'x86-64_windows') {
```

Comment:
I believe the request was going to be that the single node will be labelled ci.role.test.interactive so this logic will not work... (we are not appending a label, we are changing it so that machine is no longer being used for regular testing). To confirm with @steelhead31

Reply:
The single node label for the one windows tck automation machine is now set up, and is [ci.role.test.interactive](https://ci.eclipse.org/temurin-compliance/label/ci.role.test.interactive)